assets/sass/components/global/_googlesitekit-publisher-wins.scss

```
@@ -84,6 +84,20 @@
 			display: inline-block;
 			max-width: 100%;
 		}
+
+		@media (max-width: $bp-mobileOnly ) {
+
+			svg {
+				width: 200px;
+			}
+		}
+
+		@media (min-width: $bp-desktop) {
```

We can combine these into a single media query
```suggestion
		@media (max-width: $bp-mobileOnly ), (min-width: $bp-desktop) {
```